From 0e72d16425bf2f7a8edd3d74ae07f8cf8ff6cf06 Mon Sep 17 00:00:00 2001
From: Ivan Kravets <me@ikravets.com>
Date: Mon, 16 Sep 2024 20:45:30 +0300
Subject: [PATCH] Update bottle to 0.13.*

---
 platformio/dependencies.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/platformio/dependencies.py b/platformio/dependencies.py
index 5fd40a1d..03579077 100644
--- a/platformio/dependencies.py
+++ b/platformio/dependencies.py
@@ -30,7 +30,7 @@ def get_core_dependencies():
 
 def get_pip_dependencies():
     core = [
-        "bottle == 0.12.*",
+        "bottle == %s" % ("0.12.*" if PY36 else "0.13.*"),
         "click >=8.0.4, <9",
         "colorama",
         "marshmallow == 3.*",
-- 
2.46.0

