From 145142ea6c1d97c92a62999d424c0dd712fbb9f5 Mon Sep 17 00:00:00 2001
From: Ivan Kravets <me@ikravets.com>
Date: Mon, 24 Feb 2025 13:15:19 +0200
Subject: [PATCH] Update deps

---
 platformio/dependencies.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/platformio/dependencies.py b/platformio/dependencies.py
index 2e02fce947..b028b46d16 100644
--- a/platformio/dependencies.py
+++ b/platformio/dependencies.py
@@ -42,7 +42,7 @@ def get_pip_dependencies():
     home = [
         # PIO Home requirements
         "ajsonrpc == 1.2.*",
-        "starlette >=0.19, <0.46",
+        "starlette >=0.19, <0.47",
         "uvicorn >=0.16, <0.35",
         "wsproto == 1.*",
     ]
