From 83f780bb3cd2a7c2ff2823d54e07f1cba6eb3e5e Mon Sep 17 00:00:00 2001
From: "Benjamin A. Beasley" <code@musicinmybrain.net>
Date: Sun, 29 Dec 2024 09:56:46 -0500
Subject: [PATCH] Allow starlette versions through 0.45

---
 platformio/dependencies.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/platformio/dependencies.py b/platformio/dependencies.py
index 2489afbdf2..2c43db7b2e 100644
--- a/platformio/dependencies.py
+++ b/platformio/dependencies.py
@@ -43,7 +43,7 @@ def get_pip_dependencies():
     home = [
         # PIO Home requirements
         "ajsonrpc == 1.2.*",
-        "starlette >=0.19, <0.43",
+        "starlette >=0.19, <0.46",
         'uvicorn == 0.16.0; python_version < "3.7"',
         'uvicorn >=0.16, <0.35; python_version >= "3.7"',
         "wsproto == 1.*",
