From 64aec0b8d0904c7e091faa562fd3d25ffb098243 Mon Sep 17 00:00:00 2001
From: "Benjamin A. Beasley" <code@musicinmybrain.net>
Date: Thu, 29 May 2025 16:22:46 -0400
Subject: [PATCH] Allow starlette versions through 0.47

---
 platformio/dependencies.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/platformio/dependencies.py b/platformio/dependencies.py
index 2de0798f4e..dfb14ebe0d 100644
--- a/platformio/dependencies.py
+++ b/platformio/dependencies.py
@@ -42,7 +42,7 @@ def get_pip_dependencies():
     home = [
         # PIO Home requirements
         "ajsonrpc == 1.2.*",
-        "starlette >=0.19, <0.47",
+        "starlette >=0.19, <0.48",
         "uvicorn >=0.16, <0.35",
         "wsproto == 1.*",
     ]
