From 5548197a749fbba981fc09ca03e8ba8ccbb1981d Mon Sep 17 00:00:00 2001
From: Ivan Kravets <me@ikravets.com>
Date: Mon, 9 Oct 2023 11:41:22 +0300
Subject: [PATCH] Update "pyelftools" to 0.30

---
 platformio/__init__.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/platformio/__init__.py b/platformio/__init__.py
index d8e1515b29..2ea56425bb 100644
--- a/platformio/__init__.py
+++ b/platformio/__init__.py
@@ -59,7 +59,7 @@
     "click >=8.0.4, <=8.2",
     "colorama",
     "marshmallow == 3.*",
-    "pyelftools == 0.29",
+    "pyelftools == 0.30",
     "pyserial == 3.5.*",  # keep in sync "device/monitor/terminal.py"
     "requests == 2.*",
     "semantic_version == 2.10.*",
