From 6cf8b8172feb33fc2cb59b83f3bb29d44db8922f Mon Sep 17 00:00:00 2001
From: Ivan Kravets <me@ikravets.com>
Date: Tue, 8 Jul 2025 17:45:14 +0300
Subject: [PATCH] Update deps

---
 platformio/dependencies.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/platformio/dependencies.py b/platformio/dependencies.py
index 2de0798f4e..5f202741cd 100644
--- a/platformio/dependencies.py
+++ b/platformio/dependencies.py
@@ -42,8 +42,8 @@ def get_pip_dependencies():
     home = [
         # PIO Home requirements
         "ajsonrpc == 1.2.*",
-        "starlette >=0.19, <0.47",
-        "uvicorn >=0.16, <0.35",
+        "starlette >=0.19, <0.48",
+        "uvicorn >=0.16, <0.36",
         "wsproto == 1.*",
     ]
 
