From a4276b4ea63e8701cfa9c05d820d5dfe61c17409 Mon Sep 17 00:00:00 2001
From: Ivan Kravets <me@ikravets.com>
Date: Tue, 15 Oct 2024 22:43:13 +0300
Subject: [PATCH] Update dependencies.py

---
 platformio/dependencies.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/platformio/dependencies.py b/platformio/dependencies.py
index e358453383..d946b91c75 100644
--- a/platformio/dependencies.py
+++ b/platformio/dependencies.py
@@ -43,7 +43,7 @@ def get_pip_dependencies():
     home = [
         # PIO Home requirements
         "ajsonrpc == 1.2.*",
-        "starlette >=0.19, <0.41",
+        "starlette >=0.19, <0.42",
         'uvicorn == 0.16.0; python_version < "3.7"',
         'uvicorn >=0.16, <0.31; python_version >= "3.7"',
         "wsproto == 1.*",
