From ac6d94860b1d742a721969625e0bd1b2ce483e92 Mon Sep 17 00:00:00 2001
From: Ivan Kravets <me@ikravets.com>
Date: Fri, 1 Dec 2023 15:55:36 +0200
Subject: [PATCH] Update deps

---
 platformio/__init__.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/platformio/__init__.py b/platformio/__init__.py
index 9a8932bc94..05168327c2 100644
--- a/platformio/__init__.py
+++ b/platformio/__init__.py
@@ -67,7 +67,7 @@
 ] + [
     # PIO Home requirements
     "ajsonrpc == 1.2.*",
-    "starlette >=0.19, <0.32",
+    "starlette >=0.19, <0.33",
     "uvicorn >=0.16, <0.24",
     "wsproto == 1.*",
 ]
