diff -Naur platformio-core-6.1.18-original/platformio/dependencies.py platformio-core-6.1.18/platformio/dependencies.py
--- platformio-core-6.1.18-original/platformio/dependencies.py	2025-09-14 07:52:01.334346909 +0100
+++ platformio-core-6.1.18/platformio/dependencies.py	2025-09-14 07:52:29.246365210 +0100
@@ -42,7 +42,7 @@
     home = [
         # PIO Home requirements
         "ajsonrpc == 1.2.*",
-        "starlette >=0.19, <0.48",
+        "starlette >=0.19, <0.49",
         "uvicorn >=0.16, <0.36",
         "wsproto == 1.*",
     ]
