From 4109cb6d8fe3e0b1671c29214ee2418d34f6a6d6 Mon Sep 17 00:00:00 2001
From: Ben Beasley <code@musicinmybrain.net>
Date: Thu, 25 Jul 2024 07:33:32 -0400
Subject: [PATCH 1/2] Allow Starlette 0.38.1 (#4953)

---
 platformio/dependencies.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/platformio/dependencies.py b/platformio/dependencies.py
index 5fd40a1d..1e4ba1d7 100644
--- a/platformio/dependencies.py
+++ b/platformio/dependencies.py
@@ -44,7 +44,7 @@ def get_pip_dependencies():
     home = [
         # PIO Home requirements
         "ajsonrpc == 1.2.*",
-        "starlette >=0.19, <0.38",
+        "starlette >=0.19, <0.39",
         "uvicorn %s" % ("== 0.16.0" if PY36 else ">=0.16, <0.30"),
         "wsproto == 1.*",
     ]
-- 
2.46.0


From c9bc00259b75d060029de48851c91f8b0ab8b93a Mon Sep 17 00:00:00 2001
From: Ivan Kravets <me@ikravets.com>
Date: Mon, 23 Sep 2024 15:33:06 +0300
Subject: [PATCH 2/2] Update deps

---
 platformio/dependencies.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/platformio/dependencies.py b/platformio/dependencies.py
index 1e4ba1d7..9987ded5 100644
--- a/platformio/dependencies.py
+++ b/platformio/dependencies.py
@@ -44,7 +44,7 @@ def get_pip_dependencies():
     home = [
         # PIO Home requirements
         "ajsonrpc == 1.2.*",
-        "starlette >=0.19, <0.39",
+        "starlette >=0.19, <0.40",
         "uvicorn %s" % ("== 0.16.0" if PY36 else ">=0.16, <0.30"),
         "wsproto == 1.*",
     ]
-- 
2.46.0

