diff -Naur a/platformio/commands/upgrade.py b/platformio/commands/upgrade.py
--- a/platformio/commands/upgrade.py	2024-01-12 05:52:01.000000000 -0500
+++ b/platformio/commands/upgrade.py	2024-01-12 10:35:51.600123380 -0500
@@ -33,15 +33,14 @@
 
 
 @click.command("upgrade", short_help="Upgrade PlatformIO Core to the latest version")
-@click.option("--dev", is_flag=True, help="Use development branch")
-@click.option("--only-dependencies", is_flag=True)
 @click.option("--verbose", "-v", is_flag=True)
-def cli(dev, only_dependencies, verbose):
-    if only_dependencies:
-        return upgrade_pip_dependencies(verbose)
-
+def cli(verbose):
     update_core_packages()
 
+    # Short-circuit upgrade logic, PlatformIO is managed by DNF in Fedora
+    click.echo("PlatformIO Core is managed by DNF in Fedora")
+    return
+
     if not dev and __version__ == get_latest_version():
         return click.secho(
             "You're up-to-date!\nPlatformIO %s is currently the "
diff -Naur a/platformio/maintenance.py b/platformio/maintenance.py
--- a/platformio/maintenance.py	2024-01-12 05:52:01.000000000 -0500
+++ b/platformio/maintenance.py	2024-01-12 09:23:58.795264113 -0500
@@ -221,6 +221,9 @@
     # Update PlatformIO Core packages
     update_core_packages()
 
+    # Short-circuit upgrade logic, PlatformIO is managed by DNF in Fedora
+    return
+
     latest_version = get_latest_version()
     if pepver_to_semver(latest_version) <= pepver_to_semver(__version__):
         return
