diff --git a/platformio/dependencies.py b/platformio/dependencies.py
index b36b42b5..915e7dc1 100644
--- a/platformio/dependencies.py
+++ b/platformio/dependencies.py
@@ -31,7 +31,7 @@ def get_pip_dependencies():
         "bottle == 0.13.*",
         "click >=8.0.4, <8.4",  # click 9.0 removes 'protected_args' attribute
         "colorama",
-        "marshmallow == 3.*",
+        "marshmallow >=3, <5",
         "pyelftools >=0.27, <1",
         "pyserial == 3.5.*",  # keep in sync "device/monitor/terminal.py"
         "requests%s == 2.*" % ("[socks]" if is_proxy_set(socks=True) else ""),
diff --git a/platformio/package/manifest/schema.py b/platformio/package/manifest/schema.py
index 94e3c6a8..006120b2 100644
--- a/platformio/package/manifest/schema.py
+++ b/platformio/package/manifest/schema.py
@@ -232,7 +232,7 @@ class ManifestSchema(BaseSchema):
     )
 
     @validates("version")
-    def validate_version(self, value):
+    def validate_version(self, value, **_):
         try:
             value = str(value)
             assert "." in value
@@ -249,7 +249,7 @@ class ManifestSchema(BaseSchema):
             ) from exc
 
     @validates("license")
-    def validate_license(self, value):
+    def validate_license(self, value, **_):
         try:
             spdx = self.load_spdx_licenses()
         except requests.exceptions.RequestException as exc:
